### ISSUE TITLE
Cannot resize MainHeader Component

### ISSUE THREAD

**Issue by ruucm** [harbor-school/HSDS#1]
Please share your thought of how I change this MainHeader component size

@oneybee

https://www.loom.com/share/02f1aec1dc094b7ba514a9600f5f19f9

**Comment by oneybee** [comment]
Update component, you can use styled component and can change width, height size.

const Wrap = styled.div`
  width: 300px;
  height: 800px;
  background: red;
`
const IndexPage = () => {
  return (
       <Wrap>
          <MainHeader></MainHeader>
        </Wrap>
)}```

**Comment by oneybee** [comment]
Update component, you can use styled component and can change width, height size.

```
const Wrap = styled.div`
  width: 300px;
  height: 800px;
  background: red;
`
const IndexPage = () => {
  return (
       <Wrap>
          <MainHeader></MainHeader>
        </Wrap>
)}
```

**Comment by ruucm** [comment]
@oneybee

**Comment by oneybee** [comment]
https://github.com/harbor-school/HSDS/pull/5

**Comment by ruucm** [comment]
@oneybee

**Comment by ruucm** [comment]
@oneybee 

Components should be resized itself without other components 

Think about how many `<Wrap>` needed for using your components

**Comment by oneybee** [comment]
Fix MainHeader component resize issue. I think we can use like below.

```
        <MainHeader
          width="100"
          height="500"
        />
```

**Comment by ruucm** [comment]
@oneybee 

👍 I merged #9 and closed this issue
